src/pages/AboutPage.js

```diff
@@ -117,6 +118,37 @@ const AboutPage = () => (
     <hr />
 
     <Row className="about-align-center">
+      <Col className="about-order1">
+        <h2>Build for Black Lives in the News</h2>
+        <p> Check out these latest news articles featuring Build for Black Lives to learn
+            more about the motivation and vision behind this platform.
+        </p>
+        <p className="about-list">
+          <p>
+
+            <a href="https://news.northeastern.edu/2020/08/10/can-technology-reduce-racial-inequity-build-for-black-lives-a-student-built-platform-could-help/">
+              Can Technology Reduce Racial Inequality?
+            </a>
+            &nbsp;-
+             <span className="font-italic"> News@Northeastern</span>
+          </p>
+          <p>
+            <a href="https://undergraduatenews.khoury.northeastern.edu/2020/07/31/build-for-black-lives/">
+              Undergraduate News: Build for Black Lives
+            </a>
+            &nbsp;-
```

Comment:
I think it's nicer to do `{ -}` which includes the space rather than using `&nbsp;` and also I'd copy in an actual en or em dash rather than a hyphen